cmd/docs.go

```diff
@@ -0,0 +1,102 @@
+package cmd
+
+import (
+	"bytes"
+	"fmt"
+	"os"
+	"os/exec"
+	"strings"
+
+	"github.com/TouchBistro/goutils/fatal"
+	"github.com/TouchBistro/tb/config"
+	"github.com/TouchBistro/tb/docker"
+	"github.com/TouchBistro/tb/util"
+	"github.com/pkg/errors"
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+var docsCmd = &cobra.Command{
+	Use:   "docs <service-name> (Experimental)",
+	Args:  cobra.ExactArgs(1),
+	Short: "Opens link to API docs for a given service",
+	Long: `Opens link to API docs for a given service.
+
+	Example:
+	  tb docs core`,
```

Comment:
Nit remove the indentation here, it is taken literally and makes the help look weird